qtconsole/completion_widget.py

```diff
@@ -116,7 +116,8 @@ def show_items(self, cursor, items, prefix_length=0):
             else:
                 list_item = QtWidgets.QListWidgetItem()
                 list_item.setData(QtCore.Qt.UserRole, item)
-                list_item.setText(item.split('.')[-1])
+                # Need to split to only show last element of a dot completion
+                list_item.setText(item.split(".")[-1])
```

Comment:
why change the quote type?

Reply:
Ups my bad, by mistake I changed this line in the first commit and when I re-added it changed the quote type. Would you like better the `"` instead of the `'`? (in terms of functionality I think they work the same in this case).